packages/web/components/Quest/QuestTile.tsx

```diff
@@ -37,37 +36,34 @@ export const QuestTile: React.FC<Props> = ({ quest }) => {
     : '';
 
   return (
-    <MetaTile>
-      <Box
-        bgImage={`url(${BackgroundImage})`}
-        bgSize="cover"
-        bgPosition="center"
-        position="absolute"
-        top={0}
-        left={0}
-        w="100%"
-        h="3.5rem"
-      />
-      <Flex justify="center" mb={4}>
-        <Avatar
-          size="lg"
-          src={optimizedImage('logoURL', quest.guild.logo)}
-          name={quest.guild.name}
-        />
-      </Flex>
-
-      <MetaTileHeader>
-        <VStack>
-          <MetaLink as={`/quest/${quest.id}`} href="/quest/[id]">
+    <Link
+      role="group"
+      _hover={{ textDecoration: 'none' }}
+      href={`/quest/${quest.id}`}
+    >
+      <MetaTile height="full" width="full">
+        <MetaTileHeader>
+          <SquareImage src={BackgroundImage} />
+          <Flex px={3} w="full" pos="absolute" bottom={-6} zIndex={1}>
             <Heading
-              size="md"
+              size="lg"
               color="white"
-              fontFamily="body"
+              bgColor="rgba(255, 255, 255, 0.06)"
```

Comment:
I've been using `whiteGloss.50` to represent white with an opacity of 0.05, but I've just noticed there's already a `whiteAlpha` color.

Ok, [found it](https://chakra-ui.com/docs/styled-system/theme#alphas). Your color should be:

```suggestion
              bgColor="whiteAlpha.100"
```